firefox-ios/firefox-ios-tests/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -423,17 +424,14 @@ class BaseTestCase: XCTestCase {
         navigator.goto(SettingsScreen)
         navigator.goto(DisplaySettings)
         sleep(3)
-        if !app.switches["SystemThemeSwitchValue"].exists {
+        if !app.navigationBars["Appearance"].exists {
             navigator.goto(DisplaySettings)
         }
-        mozWaitForElementToExist(app.switches["SystemThemeSwitchValue"])
-        if (app.switches["SystemThemeSwitchValue"].value as? String) == "1" {
-            navigator.performAction(Action.SystemThemeSwitch)
-        }
+        mozWaitForElementToExist(app.navigationBars["Appearance"])
```

Comment:
Let me see if full functional tests that are related the dark/light themes are affected.